### ISSUE TITLE
Add "low-data" constraint resources

### ISSUE THREAD

**Issue by pmbaumgartner** [RTIInternational/nlp#1]
Since we usually work in environments where labeled data comes at a premium, we could explore some data augmentation / data labeling / selection approaches that might reduce the need for labeled data. A few things come to mind:

- **E-Stitchup (and more generally the idea of "label smoothing" / "soft labels"** 
  - C. R. Wolfe and K. T. Lundgaard, “E-Stitchup: Data Augmentation for Pre-Trained Embeddings,” arXiv:1912.00772 [cs, stat], Oct. 2020, Accessed: Nov. 14, 2021. [Online]. Available: http://arxiv.org/abs/1912.00772
- **Weak Supervision w/ Snorkel:** 
  - A. Ratner, S. H. Bach, H. Ehrenberg, J. Fries, S. Wu, and C. Ré, “Snorkel: rapid training data creation with weak supervision,” Proc. VLDB Endow., vol. 11, no. 3, pp. 269–282, Nov. 2017, doi: 10.14778/3157794.3157797.
- **Data Augmentation**
  - D. Tamming, “Data Augmentation For Text Classification Tasks”, Accessed: Nov. 10, 2021. [Online]. Available: https://core.ac.uk/reader/344915466
  - V. Marivate and T. Sefara, “Improving Short Text Classification Through Global Augmentation Methods,” 2020, pp. 385–399. doi: 10.1007/978-3-030-57321-8_21.
- **Submodular Selection**
  -  J. Schreiber, J. Bilmes, and W. S. Noble, “apricot: Submodular selection for data summarization in Python,” Journal of Machine Learning Research, vol. 21, no. 161, pp. 1–6, 2020, [Online]. Available: http://jmlr.org/papers/v21/19-467.html
